compute/rn/release-information/release-notes-22-06-update1.adoc

```diff
@@ -0,0 +1,53 @@
+== 22.06 Update 1 Release Notes
+
+The following table outlines the release particulars:
+
+[cols="1,4"]
+|===
+|Build
+|22.06.197
+
+|Code name
+|Kepler, 22.06 Update 1
+//Tentative date
+|Release date
+|Jul 27, 2022
+
+|Type
+|Maintenance release
+
+|SHA-256 digest
+|5aa618314e176d03e559e58d2eba50959365cdc145cba99f5d47d90737d233bf
+|===
+
+// Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+//
+// LINK
+
+=== Improvements, fixes, and performance enhancements
+//GithubIssue PCSUP-issue
+// #39791 #PCSUP-10025 On-prem
+* Fixed the broken pipe error when https://prisma.pan.dev/api/cloud/cwpp/images#operation/get-images-download[downloading a large image CSV]. The error was fixed by extending the HTTP client timeout value.
```

Comment:
@Pubs-MV we need to specify that this error occurred only for secondary consoles when using Projects. This is relevant for Self-hosted deployments with projects.